src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -69,6 +69,7 @@ export interface ISearchInterfaceOptions {
   allowQueriesWithoutKeywords?: boolean;
   responsiveMediumBreakpoint?: number;
   responsiveSmallBreakpoint?: number;
+  closeModalBox?: boolean;
```

Comment:
I see the problem, but I would not pass it as an option of the search interface. This is not something that "normal" people will want to interact with, and adding an option globally like that should always be a "last resort" kind of thing.

Could we instead have a method on the query controller that you could use to switch a flag or something ?

It should also be properly tested, which it is not, currently.

Reply:
Would you mind if i inject ModalBox instead of just using it as an import ? (Was blocking my UTs)

Reply:
Also, a public method is not a bit dirty compared to an option ?
The option is also already public and can be passed to executeQuery.
Thus, put it as a component option can make sense ?